apps/build-onchain-apps/contracts/.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
+on: workflow_dispatch
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
```

Comment:
Wait, we can have `.github` files in the different part of the app, that's neat!!
Ok I am going to move the other as well ✨

Reply:
Maybe better question, would it be better to have this file as `apps/build-onchain-apps/.github/~~~`???
So all Workflow related to `apps/build-onchain-apps` are all in the same place?

Reply:
Ok, I actually give it a try, and it seems can only be one `.github` and collection of workflows at the root of the project. So I think we need to move this to the way top to have it working.

Reply:
Thank you! For pointing this out, I will fix this. 

It's still a draft PR though, I still need to get some stuff working, I will let you know once it's done and feel free to review it again then 😄